### PR TITLE
Automatically add quotes to `content` values

### PR DESCRIPTION
**Pull request by markdalgleish**
See changeset and tests for details.

### CODE REVIEW

File: packages/css/src/transformCss.test.ts [hunk 1]
```diff
@@ -548,6 +548,219 @@ describe('transformCss', () => {
...
+      ".testClass {
+        content: \\"hello\\";
+      }
+      .testClass {
+        content: \\"there\\";
+      }"
+    `);
+  });
+
+  it('should handle content with fallacks', () => {
```
Comment (michaeltaranto) [inline packages/css/src/transformCss.test.ts:601]:
```suggestion
  it('should handle content with fallbacks', () => {
```

File: packages/css/src/transformCss.test.ts [hunk 2]
```diff
@@ -548,6 +548,219 @@ describe('transformCss', () => {
...
+                '._19 &': { content: 'revert' },
+                '._20 &': { content: 'unset' },
+              },
+            },
+          },
+        ],
+      }).join('\n'),
+    ).toMatchInlineSnapshot(`
+      ".testClass {
+        content: \\"normal\\";
```
Comment (mattcompiles) [inline packages/css/src/transformCss.test.ts:699]: Is this supposed to be quoted?

File: packages/css/src/transformCss.ts [hunk 3]
```diff
@@ -215,6 +219,30 @@ class Stylesheet {
...
+    if (typeof content === 'undefined') {
+      return rest;
+    }
+
+    // Handle fallback arrays:
+    const contentArray = Array.isArray(content) ? content : [content];
+
+    return {
+      content: contentArray.map((value) =>
+        // This logic was adapted from Stitches :)
```
Comment (mattcompiles) [inline packages/css/src/transformCss.ts:232]: 👏

File: packages/css/src/transformCss.test.ts (hunk 2 above)
Comment (markdalgleish) [reply packages/css/src/transformCss.test.ts]: Whoops, pushing a fix.

File: packages/css/src/transformCss.test.ts [hunk 4]
```diff
@@ -548,6 +548,219 @@ describe('transformCss', () => {
...
+        content: normal;
+      }
+      ._01 .testClass {
+        content: none;
+      }
+      ._02 .testClass {
+        content: url(\\"http://www.example.com/test.png\\");
+      }
+      ._03 .testClass {
+        content: \\"linear-gradient(#e66465, #9198e5)\\";
```
Comment (mattcompiles) [inline packages/css/src/transformCss.test.ts:708]: This this is a mis-quote 😝

Comment (markdalgleish) [reply packages/css/src/transformCss.test.ts]: Pushed a fix. We didn't match on hyphens in function names.

**Review by mattcompiles — APPROVED** [review]
Awesome work. Also thanks to the Stitches team again as well 😅

**Pull request merged**